remove-disabled-formulae/main.rb

```diff
@@ -36,4 +36,7 @@ def git(*args)
   git '-C', tap_dir.to_s, 'commit', formula.path.to_s, '--message', "#{formula.name}: remove formula", '--quiet'
 end
 
-puts '::set-output name=formulae-removed::true'
+github_output = ENV.fetch("GITHUB_OUTPUT") { raise "GITHUB_OUTPUT is not defined" }
```

Comment:
This will not work, we use `brew ruby` and need to pass $GITHUB_OUTPUT as an argument to the script (the same about other ruby scripts)

https://github.com/Homebrew/actions/blob/bea9c11027209b0a972534bff1fb0b411d51a0ed/remove-disabled-formulae/action.yml#L14

Reply:
I don't think we should be relying on `brew ruby` calling `Formula` `GITHUB_OUTPUT`  like this in this repository. Feels like a `brew disabled` or command or `brew info --json --disabled` or something in Homebrew/brew consumed by plain Ruby or Bash would be a better fit.